topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.md

```diff
@@ -117,50 +115,64 @@ After you confirm that all the data and metadata looks ok, you can go ahead and
 
 > ### {% icon hands_on %} Hands-on: add metadata interactively and submit a single sequence to ENA
 >
-> 1. {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %}:
->    - *"Submit to test ENA server?"*: `yes`
+> 1. {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.6.1) %}:
```

Comment:
Here it is useful, in comparison to the previous, here they're likely to click on it because they need it now.

Reply:
Done, I kept them in the hands on pieces!